lib/api/v1/controllers/keywords.go

```diff
@@ -35,3 +37,31 @@ func (KeywordsController) Upload(ctx *gin.Context) {
 
 	jsonhelpers.RenderJSON(ctx, http.StatusOK, nil)
 }
+
+func (KeywordsController) List(ctx *gin.Context) {
```

Comment:
```suggestion
func (KeywordsController) Index(ctx *gin.Context) {
```

There is a sort of [CRUD convention](https://medium.com/@atharvakulkarniamk/restful-routing-mapping-rest-to-crud-dbb2f32f748c), there is no obligation to follow it, but as most teammates (and web frameworks) know and apply it, it's more effective to describe what it actually does 👍